go.mod

```diff
@@ -0,0 +1,26 @@
+module github.com/codacy/codacy-engine-golang-seed
+
+go 1.13
+
+require (
+	github.com/360EntSecGroup-Skylar/goreporter v0.0.0-20180902115603-df1b20f7c5d0 // indirect
+	github.com/facebookgo/inject v0.0.0-20180706035515-f23751cae28b // indirect
```

Comment:
uhmmmm, there are a lot of indirect dependencies, can you verify why?

Reply:
By removing the unused dependencies, some indirect dependencies have been cleaned up